paradise.dme

```diff
@@ -611,7 +611,6 @@
 #include "code\game\machinery\shieldgen.dm"
 #include "code\game\machinery\Sleeper.dm"
 #include "code\game\machinery\slotmachine.dm"
-#include "code\game\machinery\snow_machine.dm"
```

Comment:
you need to make sure your thing is up to date and make sure these files have a checkmark when you compile

Reply:
![image](https://user-images.githubusercontent.com/20497797/50256761-e233d880-03c5-11e9-91d7-f0226b980a49.png)

example